drivers/clk/ralink/Kconfig

```diff
@@ -9,3 +9,10 @@ config CLK_MT7621
 	select MFD_SYSCON
 	help
 	  This driver supports MediaTek MT7621 basic clocks.
+
+config CLK_RALINK
```

Comment:
Ok I think it'll be less work for both of us if we start naming things mtmips from now on. I'm already working on some patches on my own to change the architecture code, kernel options, etc. to mtmips. It'd be better if new things like these are already called mtmips.

That said, please change this to CLK_MTMIPS.

Reply:
Sure